sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2025-2026.bpmn

```diff
@@ -1859,7 +1859,7 @@ TotalChildcareDependants: 11yo and younger</bpmn:documentation>
           <zeebe:output source="=if studentDataDependants = null then 0 else&#10;count(&#10; studentDataDependants[&#10; // Dependants who are aged between 19-22 years (i.e until they are 23 years old) and not declared on taxes due to disability&#10; date(item.dateOfBirth) &#60;= (date(offeringStudyStartDate) - duration(&#34;P19Y&#34;))&#10; and date(item.dateOfBirth) &#62; (date(offeringStudyStartDate) - duration(&#34;P23Y&#34;))&#10; and item.attendingPostSecondarySchool = &#34;yes&#34;&#10; and item.declaredOnTaxes = &#34;no&#34; ]&#10;)" target="calculatedDataDependantPostSecondaryQuantity" />
           <zeebe:output source="=if studentDataDependants = null then 0 else&#10;  count(&#10;studentDataDependants[&#10;// 0-18 years of age: eligible.&#10;date(item.dateOfBirth) &#62;= (date(offeringStudyStartDate) - duration(&#34;P18Y&#34;))&#10;// 19-22 years of age: post-sec must be Yes or Tax is Yes.&#10;or (&#10;  date(item.dateOfBirth) &#60; (date(offeringStudyStartDate) - duration(&#34;P18Y&#34;))&#10;  and date(item.dateOfBirth) &#62;= (date(offeringStudyStartDate) - duration(&#34;P22Y&#34;))&#10;  and (item.attendingPostSecondarySchool = &#34;yes&#34; or item.declaredOnTaxes = &#34;yes&#34;)&#10;)&#10;// &#62; 22 years: eligible if Tax is Yes.&#10;or (&#10;  date(item.dateOfBirth) &#60; (date(offeringStudyStartDate) - duration(&#34;P22Y&#34;))&#10;  and item.declaredOnTaxes = &#34;yes&#34;&#10;)])" target="calculatedDataTotalEligibleDependants" />
-          <zeebe:output source="=if studentDataDependants = null then 0 else&#10;count(studentDataDependants[&#10;      // Dependant who is between 0 and 11 years on study start date.&#10;      // i.e. until the day(study start date) before their 12th birthday the dependant &#10;      // is considered to be in this category.&#10;      date(item.dateOfBirth) &#62; (date(offeringStudyStartDate) - duration(&#34;P12Y&#34;))&#10;      and date(item.dateOfBirth) &#60;= date(offeringStudyEndDate)]&#10;  )" target="calculatedDataTotalChildcareDependants" />
+          <zeebe:output source="=if studentDataDependants = null then 0 else&#10;count(studentDataDependants[&#10;      // Dependant who is between 0 and 11 years on study start date.&#10;      // i.e. until the day(study start date) before their 12th birthday the dependant &#10;      // is considered to be in this category.&#10;      date(item.dateOfBirth) &#62; (date(offeringStudyStartDate) - duration(&#34;P12Y&#34;))&#10;      and date(item.dateOfBirth) &#60;= date(offeringStudyEndDate) or&#10;// Dependants who are declared on taxes due to disability and are 12 or over.&#10;      date(item.dateOfBirth) &#60;= (date(offeringStudyStartDate) - duration(&#34;P12Y&#34;))&#10;      and date(item.dateOfBirth) &#60;= date(offeringStudyEndDate)&#10;      and item.declaredOnTaxes = &#34;yes&#34;]&#10;  )" target="calculatedDataTotalChildcareDependants" />
```

Comment:
Can we review this logic? I have a feeling there are some parentheses missing to group some conditions, or am I missing something?
Please take a look at the `calculatedDataTotalEligibleDependants` and how the parentheses create the "grouped conditions".